react/components/Charts/hooks/useChart.ts

```diff
@@ -0,0 +1,8 @@
+import { commonDefaultProps } from '../commonProps'
+
+const useChart = (userProps: ChartProps)  => {
```

Comment:
This hook does not seem that useful, you could just import the `commonDefaultProps` on the desired component. And you would do something like:
```tsx
// Component File
import { commonDefaultProps } from 'constants'

const Component: FC<Props> = props => { ... }

Component.defaultProps = {
  ...commonDefaultProps
}

interface Props  extends ChartProps {
  // otherProps
}
```

Reply:
Because this object sets some props that aren't editables on the component, and used in each `recharts` component. But i agree with you that the hook is not useful, i can import only the object, and merge with the props defined by the user.